dash-sdk-java/src/main/swig/generics/lists.i

```diff
@@ -69,7 +69,7 @@
         return $null;
     }
     for (uintptr_t i = 0; i < $1->count; ++i) {
-        jobject elementObj = jenv->NewObject(valueClass, valueConstructor, $1->values[i], false);
+        jobject elementObj = jenv->NewObject(valueClass, valueConstructor, (jlong) $1->values[i], false);
```

Comment:
This change is for 32-bit support. `jlong` size depends on 32-bit or 64-bit architecture.